recipes/spirv-tools/all/conanfile.py

```diff
@@ -0,0 +1,86 @@
+from conans import ConanFile, tools, CMake
+import os
+
+
+class SpirvtoolsConan(ConanFile):
+    name = "spirv-tools"
+    homepage = "https://github.com/KhronosGroup/SPIRV-Tools/"
+    description = "SPIRV-Tools "
+    topics = ("conan", "spirv", "spirv-v", "vulkan", "opengl", "opencl", "hlsl", "khronos")
+    url = "https://github.com/conan-io/conan-center-index"
+    settings = "os", "compiler", "arch", "build_type"
+    exports_sources = ["CMakeLists.txt"]
+    license = "Apache-2.0"
+    generators = "cmake"
+
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "c_only": [True, False]
+    }
+    default_options = {
+        "shared": True,
+        "fPIC": True,
+        "c_only": False
+    }
+
+    def requirements(self):
+        self.requires.add("spirv-headers/1.5.1")
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = "SPIRV-Tools-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def configure(self):
+        if self.settings.os == 'Windows':
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+
+        # Required by the project's CMakeLists.txt
+        cmake.definitions["SPIRV-Headers_SOURCE_DIR"] = self.deps_cpp_info["spirv-headers"].rootpath
```

Comment:
You could add an option in order to set `SPIRV_SKIP_EXECUTABLES` (OFF by default).

Reply:
I can add this in, but is there any reason to NOT have the executable when building a dependency? You're not forced to use them (eg: by linking), you simply don't use them. All it does is save a few seconds of compiling time. Adding this extra option causes an extra set of permutations.

Reply:
Yes, please, leave the executables there, someone might want to use this packages as a build_requires, or they may install it using a virtualenv generator just to use those executables.